pennylane/tape/qnode.py

```diff
@@ -683,3 +683,40 @@ def qfunc_decorator(func):
         return update_wrapper(qn, func)
 
     return qfunc_decorator
+
+
+def make_drawing(_qnode, charset="unicode"):
+    """Make a function that draws the given _qnode.

```

Comment:
```suggestion
    """Create a function that draws the given _qnode.
```
Only minor, maybe `return` is even better. I think it is the same feeling as with "make drawing", the verb just feels imprecise.

Reply:
Feel free to ignore, since I am not native English speaking :)